project-p3-forms-master/code/style.css

```diff
@@ -0,0 +1,332 @@
+}
+
+/*** Top & Bottom Hamburger Lines ***/
+
+.hamburger-dropdown-menu .hamburger > div:before,
+.hamburger-dropdown-menu .hamburger > div:after {
+  content: '';
+  position: absolute;
+  z-index: 1;
+  top: -10px;
+  width: 100%;
+  height: 2px;
+  background: inherit;
+}
+
+.hamburger-dropdown-menu .hamburger > div:after {
+  top: 10px;
+}
+
+/*** Toggler animate ***/
+
+.hamburger-dropdown-menu .toggler:checked + .hamburger > div {
+  transform: rotate(135deg);
+}
+
+/*** Turn lines into X ***/
+
+.hamburger-dropdown-menu .toggler:checked + .hamburger > div:before,
+.hamburger-dropdown-menu .toggler:checked + .hamburger > div:after {
+  top: 0;
+  transform: rotate(90deg);
+}
+
+/*** Show dropdown Menu ***/
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu {
+  visibility: visible;
+}
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu > div {
+  transform: scale(1);
+  transition-duration: 0.75s;
+}
+
+.hamburger-dropdown-menu .toggler:checked ~ .menu > div > div {
+  opacity: 1;
+  transition:  opacity 0.4s ease 0.4s;
+}
+
+.hamburger-dropdown-menu .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100;
+  visibility: hidden;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.hamburger-dropdown-menu .menu > div {
+  background: #24292d;
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  align-items: flex-start;
+  justify-content: left;
+  padding-left: 20px;
+  padding-top: 40px;
+  transform: scale(0);
+  transition: all 0.4s ease;
+}
+
+.hamburger-dropdown-menu .menu > div > div {
+  text-align: left;
+  max-width: 90vw;
+  max-height: 100vh;
+  opacity: 0;
+  transition: opacity 0.4s ease;
+}
+
+.hamburger-dropdown-menu .menu > div > div > ul > li {
+  list-style: none;
+  color: #FFF;
+  font-size: 30px;
+  padding: 5px;
+}
+
+.hamburger-dropdown-menu .menu > div > div > ul > li > a {
+  color: inherit;
+  text-decoration: none;
+  transition: color 0.4s ease;
+}
+
+/*** Tablet/Desktop Menu ***/
+
+.desktop-menu {
+display: none;
+}
+
+.desktop-menu a {
+  color: #fff;
+  text-decoration: none;
+  padding: 17px 0;
+  margin-right: 16px;
+}
+
+.desktop-menu a:hover {
+  text-decoration: underline;
+}
+
+/*** About Container ***/
+
+.about-container {
+  background: #046F9E;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Essays Container ***/
+
+.essays-container {
+  background: #D3D6D7;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Letters Container ***/
+
+.letters-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
+
+/*** Newsletter Container ***/
+
+.newsletter-container {
+  background: #D3D6D7;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  padding: 10px 24px 10px 24px;
+}
```

Comment:
Since everything except color is the same for these, you could give these some common styling like this:
```
.about-container, .essays-container, .letters-container, .newsletter-container {
  //Styling that is the same for all
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  padding: 10px 24px 10px 24px;
}
```
and then writing the only thing that differs separately: 
```
.about-container {
  background: #046F9E;
}
```

Reply:
A suggestion would also be to add some more padding for these in desktop mode since it's hard to read that long lines. Something like this?
<img width="1436" alt="Skärmavbild 2020-11-26 kl  17 42 13" src="https://user-images.githubusercontent.com/51529201/100376150-4b3f1480-300f-11eb-8a84-7931585a5cd5.png">